### ISSUE TITLE
Duplicate ?xml when rendering the document

### ISSUE THREAD

**Issue by ThePlenkov** [NaturalIntelligence/fast-xml-parser#689]
Hi!

I'm testing it now using this page: https://naturalintelligence.github.io/fast-xml-parser/

I noticed that if I provide XML like this:
```
<?xml version="1.0" encoding="utf-8"?>
<test></test>
```

it generates me the JSON like this:
```
{
    "?xml": {
        "@_version": "1.0",
        "@_encoding": "utf-8"
    },
    "test": ""
}
```

now when I render XML again I have ?xml node described twice:
```
<?xml version="1.0"?>
<?xml version="1.0" encoding="utf-8"?>
<test></test>
```

and if I parse such XML of course it fails.

Could you please check why? Is it a bug of the library or just the UI issue? Thanks!

**Comment by amitguptagwl** [comment]
This is just in demo, not the library bug.

**Comment by ThePlenkov** [comment]
yes, I can confirm! it works as expected in the code. thanks. anyway - would be nice to fix the demo if possible,
